tests/test_html2docx.py

```diff
@@ -34,7 +36,11 @@ def test_html2docx(html_path, spec_path):
         runs_spec = p_spec["runs"]
         assert len(p.runs) == len(runs_spec)
         for run, run_spec in zip(p.runs, runs_spec):
-            assert run.text == run_spec["text"]
-            for attr in ("bold", "italic", "underline", "subscript", "superscript"):
+            assert run.text == run_spec.pop("text")
+            unknown = set(run_spec).difference(FONT_ATTRS)
+            assert not unknown, "Unknown attributes in {}: {}".format(
+                html_path.name, ", ".join(unknown)
```

Comment:
Should that be the `json_path.name`?

Reply:
Yes! Thanks for catching. It is now `spec_path.name` in the latest revision.